crates/env/src/engine/on_chain/ext.rs

```diff
@@ -348,7 +345,10 @@ mod sys {
             output_ptr: Ptr32Mut<[u8]>,
             output_len_ptr: Ptr32Mut<u32>,
         ) -> ReturnCode;
+    }
 
+    #[link(wasm_import_module = "__unstable__")]
+    extern "C" {
```

Comment:
This should've been removed in https://github.com/paritytech/ink/pull/1036, no? Do you need to merge `master`?

Reply:
There is still `seal_ecdsa_recover` which is unstable, but I just merged `master` and there were some conflicts to resolve because of that PR, so good to merge nontheless.